src/dpr/components/search/clientClass.mjs

```diff
@@ -1,43 +1,44 @@
+/* eslint-disable no-param-reassign */
 import { DprClientClass } from '../../DprClientClass.mjs'
 
 export default class Search extends DprClientClass {
-  static getModuleName() {
+  static getModuleName () {
     return 'search'
   }
 
   updateSearchListing (value) {
     const table = this.getElement().querySelector('.dpr-search-table').querySelector('tbody')
```

Comment:
Because the table is now provided pre-built, this means whoever uses this macro needs to know to add the `dpr-search-table` class to their table.

You could instead probably get away with just searching for a table tag, as there should only be one table within the  search component.